### DISCUSSION TITLE
how to deploy hydra in kubernate cluster

### DISCUSSION THREAD

**Discussion by edqartwizaru** [ory/hydra#3075] [Q&A] [answered]
Hi ,

please help me out with how to deploy the ory/hydra in the Kubernetes cluster

**Comment by vinckr** [comment, marked as answer]
Sorry you're having trouble. We'll need more information in order to figure out what's happening here.
Can you provide some additional information, such as what were you trying to do?
Your setup, additional log files, the software version, your configuration and any other information that you think is helpful also helps us to solve your issue more quickly!

Also see these documents: 
- https://ory.sh/docs/hydra/configure-deploy
- https://www.ory.sh/run-oauth2-server-open-source-api-security/

If you are having trouble running a Kubernetes cluster, feel free to also reach out to the [Kubernetes community](https://kubernetes.slack.com/) - they are the experts for all things k8s.
